Vienna/Sources/Parsing/RSSFeed.m

```diff
@@ -137,7 +137,8 @@ - (BOOL)initRSSFeedHeaderWithElement:(NSXMLElement *)channelElement
             (isRSSElement && [channelItemTag isEqualToString:@"pubDate"]) ||
             ([element.prefix isEqualToString:self.dcPrefix] && [channelItemTag isEqualToString:@"date"])) {
             NSString *dateString = element.stringValue;
-            self.modifiedDate = [self dateWithXMLString:dateString];
+            //publication date will be set to the current date in a later step, so we don´t set it here
```

Comment:
I end up having publication date that are later than last update date, which is counter-intuitive and probably not intended by you.